pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -723,16 +713,19 @@ void Panels::MemoryCardListPanel_Simple::UiDeleteCard(McdSlotItem& card)
 	if (card.IsFormatted)
 	{
 		wxString content;
+		wxString toPrint = Path::ToWxString(card.Filename);
 		content.Printf(
-			pxE(L"You are about to delete the formatted memory card '%s'. All data on this card will be lost!  Are you absolutely and quite positively sure?"), WX_STR(card.Filename.GetFullName()));
+			pxE( L"You are about to delete the formatted memory card '%s'. All data on this card will be lost!  Are you absolutely and quite positively sure?"
+				), WX_STR(toPrint)
```

Comment:
I feel like before the closing parenthesis wasn't the right place to break

Usually you do
```
something(
	something(a),
	something(b)
```
not
```
something(
	something(a
		), something(b)
```
right?